data/presets/presets/shop/craft.json

```diff
@@ -1,16 +1,9 @@
 {
     "icon": "fas-cut",
-    "geometry": [
-        "point",
-        "area"
-    ],
+    "geometry": ["point", "area"],
```

Comment:
It's not a big deal but please leave arrays on multiple lines if that's how they're already formatted.

Reply:
My bad I thought the lint would fix that.

Reply:
We don't autoformat the preset JSON at the moment…